doc/build-instructions/Build_Instructions_V11.md

```diff
@@ -783,6 +783,10 @@ Copy its contents to your AArch64 Linux device.
 
 :pencil: If you want a binary for the runtime environment (jre), you must run `make legacy-jre-image`, which produces a jre build in the **build/linux-aarch64-normal-server-release/images/jre** directory.
 
+:pencil: One of the images created with `make all` is the `debug-image`. This directory contains files that provide debug information for executables and shared libraries when using native debuggers.
+To use it, copy the contents of `debug-image` over the jdk before using the jdk with a native debugger.
+Another image created is the `test` image, which contains executables and native libraries required when running some functional and openjdk testing.
```

Comment:
Perhaps it would be helpful to explain how to use the files in the test image? For example, setting these environment variables:
```
LD_LIBRARY_PATH=$images/test/openj9
NATIVE_TEST_LIBS=$images/test/openj9
```

Reply:
I've pushed an update.